book.rb

```diff
@@ -1,5 +1,21 @@
+# class Book
+#   attr_accessor :title, :author
+#   attr_reader :rentals
+
+#   def initialize(title, author)
+#     @title = title
+#     @author = author
+#     @rentals = []
+#   end
+
+#   def add_rental(date, person)
+#     Rental.new(date, self, person)
+#   end
+# end
+
```

Comment:
- [ ] Impressive progress in developing the UI of your school library app. I'd like to provide a constructive suggestion to enhance your codebase further. It's crucial to maintain a clean and organized codebase, which leads me to recommend the removal of any commented-out `code` within your files.

    - In pursuit of an optimized and well-maintained code, I urge you to undertake the task of removing all commented-out `code`. This proactive step will not only improve the present state of your code but will also simplify future development and collaboration.

Happy coding! 😊🚀🍀